articles/logic-apps/quickstart-create-first-logic-app-workflow.md

```diff
@@ -31,13 +31,13 @@ Sign in to the [Azure portal](https://portal.azure.com) with your Azure account
 
 1. From the Azure home page, in the search box, find and select **Logic Apps**.
 
-   ![Find and select "Logic Apps"](./media/quickstart-create-first-logic-app-workflow/find-select-logic-apps.png)
+   ![Find and select "logic apps"](./media/quickstart-create-first-logic-app-workflow/find-select-logic-apps.png)
 
 1. On the **Logic Apps** page, select **Add**.
 
    ![Add new logic app](./media/quickstart-create-first-logic-app-workflow/add-new-logic-app.png)
 
-1. Under **Create logic app**, provide details about your logic app as shown here. After you're done, select **Create**.
+1. Under **Create logic app**, provide details about your logic app as shown below. After you’re done, select **Create**.
```

Comment:
Please refer to UI for proper formatting:

![image](https://user-images.githubusercontent.com/13206671/68044558-cf9c5800-fc94-11e9-9b01-c2811eafe7be.png)

Reply:
Not quite sure what you mean. I'm using below as it refers to the image below.
![image](https://user-images.githubusercontent.com/32169182/68045378-435c5600-fcd1-11e9-8ef0-9e533ff2c5b4.png)